### PR TITLE
Feature/fast text

### PR DESCRIPTION
**Pull request by MartinXPN**
Added support for training the model with fastText instead of GloVe.
Currently one needs to install fastText on the machine and then provide its path on preprocessing step.
Also the default version of fastText (trained on english wiki) is downloaded automatically.

### CODE REVIEW

**Comment by mahnerak** [comment]
1. Is there any evidence that fastText performs better than Glove?
2. You may consider using Gensim FastText model instead of opening subprocesses and interacting with them using standard streams.

**Comment by MartinXPN** [comment]
Actually fastText performed a bit worse than GloVe. After 60 epochs with 60hdim it reached about 50% accuracy

**Comment by jayajackson** [comment]
Dear Sir,

             I have changed the keras version.But showing the same error.


Thanking You
